tests/coverage.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -eux
+
+set -o pipefail
+
+# Script Collection CI
+# Once working, we may move this script elsewhere - gundalow
+
+
+if find tests/output/coverage/ -mindepth 1 -name '.*' -prune -o -print -quit | grep -q .; then
+    stub=""
+
+    # shellcheck disable=SC2086
+    ansible-test coverage xml -v --requirements --group-by command --group-by version ${stub:+"$stub"}
+
+    # upload coverage report to codecov.io
+    for file in tests/output/coverage/coverage=*.xml; do
```

Comment:
can you add a comment with an example of file name parsed here ?

Reply:
Good idea, added